### PR TITLE
Kumarev/update behaviors of stack resources

### PR DESCRIPTION
**Pull request by kumarev**
update behaviors of stack resources:

Workshop content for 'update behaviors of stack resources'

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: workshop/content/intermediate/operations/update-behaviors-of-stack-resources/_index.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+    Properties:
+      InstanceType: !Ref InstanceType
+      ImageId: !Ref LatestAmiId
+      Monitoring: true
+      Tags:
+        - Key: Name
+          Value: cfn-workshop-new-value
+```
+
+Save your changes to the `update-behaviors-of-stack-resources.yaml` template. Before you update the stack with your updated template, see [Update requires](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-instance.html#cfn-ec2-instance-tags) for the `tags` property: *Update requires*: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt).
```
Comment (mrinaudo-aws) [inline workshop/content/intermediate/operations/update-behaviors-of-stack-resources/_index.md:181]:
```suggestion
Save your changes to the `update-behaviors-of-stack-resources.yaml` template. Before you update the stack with your updated template, see the **Update requires** [section](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-instance.html#cfn-ec2-instance-tags) for the `Tags` property; you will notice that **Update requires**, in this case, requires [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt).
```

File: workshop/content/intermediate/operations/update-behaviors-of-stack-resources/_index.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+5. In **Prepare template**, choose **Replace** **current template**, and then choose the `update-behaviors-of-stack-resources.yaml` template mentioned earlier. When ready, choose **Next**.
+6. On the next page, choose to accept default values for `LatestAmiId`  and `InstanceType` parameters, and choose **Next**.
+7. Choose to accept default values in the **Configure stack options** page, and choose **Next**.
+8. On the **Review** page, scroll down to the bottom and choose **Update stack**.
+
+
+
+### Conclusion
+
+Congratulations! You now have now learned about the update behaviors of stack resources!
```
Comment (mrinaudo-aws) [inline workshop/content/intermediate/operations/update-behaviors-of-stack-resources/_index.md:198]:
```suggestion
Congratulations! You now have now learned the update behaviors of stack resources!
```

**Review by mrinaudo-aws — CHANGES_REQUESTED** [review]
Thank you, @kumarev !  Please review/approve suggestions, and ask for another review!

**Review by mrinaudo-aws — APPROVED** [review]
lgtm

**Review by rezabekf — CHANGES_REQUESTED** [review]
Just a small rewording of introduction. Other than that LGMT. Once my comment gets resolved, I am ready to merge.

**Comment by kumarev** [comment]
> Just a small rewording of introduction. Other than that LGMT. Once my comment gets resolved, I am ready to merge.

Done, applied the suggestions. Thanks!

**Review by rezabekf — APPROVED** [review]
Approved

**Pull request merged**